libraries/AP_HAL_ChibiOS/hwdef/omnibusf4/hwdef.dat

```diff
@@ -4,9 +4,13 @@
 
 include ../omnibusf4pro/hwdef.dat
 
+undef APJ_BOARD_ID
 undef PC8
 undef PC9
 
+# board ID for firmware load
+APJ_BOARD_ID 146
```

Comment:
the APJ_BOARD_ID must be the same for the hwdef-bl.dat and hwdef.dat for a board or it will not load correctly

Reply:
I thought I had changed them both to 146....now 1002 and title corrected....
so, in the future, I would need to request a board id, not just add one to the boards list like I attempted...correct?